docs/fuel-101/transaction-model.mdx

```diff
@@ -3,6 +3,132 @@ title: Transaction Model
 category: Fuel 101
 ---
 
-# Transaction Model
+# Transaction & Block Models
 
-Fuel's transaction model explained.
+Transactions and blocks on the Fuel network have a unique structure that is optimized for speed and scalability. Below you can find an outline of the structure of transactions and blocks on the Fuel network.
+
+- [Transaction Types](#what-are-the-different-types-of-transactions)
+- [Transaction Data](#what-data-is-included-in-a-transaction)
+- [Transaction Validation](#how-are-transactions-validated)
+- [Block Model](#what-does-the-fuel-block-model-look-like)
+
+## What are the different types of transactions?
+
+There are currently three types of transactions:
+
+<CodeImport
+  file="../fuel-specs/src/tx-format/transaction.md"
+  lang="rust"
+  lineStart="4"
+  lineEnd="8"
+/>
+
+1. **Script Transactions** include all other types of transactions that are not `Create` or `Mint` transactions.
```

Comment:
This doesn't really describe what Script is or does. I think it might make more sense to describe script as something that's executed instantly, and doesn't hold internal state, i.e. contrasting that to the contract.

Also, we will be introducing new Upgrade and Upload transactions soon, and this explation will become outdated.